code/rdb/endofperiod.q

```diff
@@ -0,0 +1,16 @@
+/-endofperiod function
+endofperiod:{[currp;nextp;data] 
+	.lg.o[`endofperiod;"Received endofperiod. currentperiod, nextperiod and data are ",(string currp),", ", (string nextp),", ", .Q.s1 data];
+	/-Obtain handle of any other running rdb's
+	h:exec w from .servers.SERVERS where proctype=`rdb,not w=0N;
+        /-Create a list of start times of rdb's found from above
+	times:@[;".proc.starttimeUTC";()]each h;
+        /-If we are the new process, exit function, do not want to writedown
```

Comment:
this comment probably needs changed, we're not writing down in the old process either, right?